cypress/e2e/photoUpdateTest.spec.js

```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress"/>
+
+import { Site } from '../pages/SiteLogIn.spec.js';
+import { EditModal } from '../pages/EditProfileModal.spec.js';
+
+describe('Photo upload test', () => {
+  const site = new Site();
+  const editModal = new EditModal();
+
+  it('photo upload check', () => {
+    site.open(),
```

Comment:
```suggestion
    site.open();
```
чому після кожної дії кома?

Reply:
так рекомендував зробити лінтер